### PR TITLE
mail notification for fully funded project

### PR DESCRIPTION
**Pull request by josipkoncurat**

### CODE REVIEW

**Comment by githinho** [comment]
@filipdujmusic this PR is still open after 22 days!

File: .gitignore [hunk 1]
```diff
@@ -100,4 +100,11 @@ typings/
...
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+/out/
```
Comment (mightymatth) [inline .gitignore:110]: All files should be always ending with an empty line.

File: grpc/mail-service.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        if (err != null) {
+            logger.error(`SUPERVISOR-QUEUE: Error while calling sendProjectFullyFunded GRPC route: %o`, err)
+            throw err
+        } else {
+            logger.info(`SUPERVISOR-QUEUE: Project full funded called on GRPC Mail Service`)
+        }
+    })
+}
+
+module.exports = { init, sendProjectFullyFunded }
```
Comment (mightymatth) [inline grpc/mail-service.js:30]: All files should be always ending with an empty line.

File: proto/mail_service.proto [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  rpc SendOrganizationInvitation(OrganizationInvitationRequest) returns (Empty);
+  rpc SendDepositRequest(DepositRequest) returns (Empty);
+  rpc SendWithdrawInfo(WithdrawInfoRequest) returns (Empty);
+  rpc SendWithdrawRequest(WithdrawRequest) returns (Empty);
+  rpc SendDepositInfo(DepositInfoRequest) returns (Empty);
+  rpc SendResetPassword(ResetPasswordRequest) returns (Empty);
+  rpc SendNewWalletMail(NewWalletRequest) returns (Empty);
+  rpc SendWalletActivated(ActivatedWalletRequest) returns (Empty);
+  rpc SendProjectFullyFunded(ProjectFullyFundedRequest) returns (Empty);
+}
```
Comment (mightymatth) [inline proto/mail_service.proto:85]: All files should be always ending with an empty line.

File: service/transaction-processor.js [hunk 4]
```diff
@@ -536,6 +538,12 @@ async function callSpecialActions(tx) {
         walletServiceGrpcClient.updateCoopRoles(coopInfo.id)
         ws.notifySubscribersForTransaction(tx)
     }
+    if (tx.type === enums.TxType.INVEST) {
+        let projectInfo = await projectService.getProjectInfoByWallet(tx.to_wallet, tx.coop_id)
```
Comment (mightymatth) [inline service/transaction-processor.js:542]: use `const` instead of `let` if the value is not meant to be changed.

**Review by mightymatth — CHANGES_REQUESTED** [review]

**Review by filipdujmusic — APPROVED** [review]

**Pull request merged**